packages/react-page-renderer/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@ima/react-page-renderer",
```

Comment:
Not that it's required, but how about this directory structure, inside the `src` folder, in order to somehow group those files by some context?

```
src
 |_ renderer
    |_ AbstractClientPageRenderer.ts 
    |_ AbstractPageRenderer.ts
    |_ ClientPageRenderer.ts
    |_ LegacyClientPageRenderer.ts
    |_ ServerPageRenderer.ts
    |_ PageRendererFactory.ts
 |_ components
    |_ AbstractComponent.ts
    |_ AbstractPureComponent.ts
    |_ ErrorBoundary.ts
    |_ ViewAdapter.ts
    |_ BlankManagedRootView.ts
 |_ PageContext.ts
 |_ componentHelpers.ts
...
```